### PR TITLE
Enh/class parachute

### PR DESCRIPTION
**Pull request by FranzYuri**
## Pull request type

Please check the type of change your PR introduces:

- [X] Code base additions (bugfix, features)
- [ ] Code maintenance (refactoring, formatting, renaming, tests)
- [ ] ReadMe, Docs and GitHub maintenance
- [ ] Other (please describe):

## Pull request checklist

Please check if your PR fulfills the following requirements, depending on the type of PR:

- ReadMe, Docs and GitHub maintenance:

  - [ ] Spelling has been verified
  - [ ] Code docs are working correctly 

- Code base maintenance (refactoring, formatting, renaming):

  - [ ] Docs have been reviewed and added / updated if needed
  - [ ] Lint (`black rocketpy`) has passed locally and any fixes were made
  - [ ] All tests (`pytest --runslow`) have passed locally

- Code base additions (for bug fixes / features):

  - [ ] Tests for the changes have been added
  - [ ] Docs have been reviewed and added / updated if needed
  - [X] Lint (`black rocketpy`) has passed locally and any fixes were made
  - [X] All tests (`pytest --runslow`) have passed locally

## What is the current behavior?

Currently parachutes are types inside class Parachute.

## What is the new behavior?

Now, there is a separated class for parachutes.

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

## Other information

Enter text here...

### CODE REVIEW

File: rocketpy/Parachute.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+from .Function import Function
+
+
+class Parachute:
+    """Keeps parachute information.
+
+    Attributes
+    ----------
+Parachute attributes:
```
Comment (Gui-FernandesBR) [inline rocketpy/Parachute.py:31]: Ident correction @FranzYuri

File: rocketpy/Parachute.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        self.cleanPressureSignal = []
+        self.noiseBias = noise[0]
+        self.noiseDeviation = noise[1]
+        self.noiseCorr = (noise[2], (1 - noise[2] ** 2) ** 0.5)
+        alpha, beta = self.noiseCorr
+        self.noiseFunction = lambda: alpha * self.noiseSignal[-1][
+            1
+        ] + beta * np.random.normal(noise[0], noise[1])
+
+
```
Comment (Gui-FernandesBR) [inline rocketpy/Parachute.py:133]: Could you define a return? @FranzYuri

File: rocketpy/Flight.py [hunk 3]
```diff
@@ -21,6 +21,7 @@
 from matplotlib import cm
 
 from .Function import Function
+from .Parachute import Parachute
```
Comment (Gui-FernandesBR) [inline rocketpy/Flight.py:24]: I believe it's not necessary here

**Review by Gui-FernandesBR — CHANGES_REQUESTED** [review]
Awesome! It was a both simple and good pull request, I liked it. I have some commentaries asking for changes, hope you enjoy it in order to improve your code

**Comment by Gui-FernandesBR** [comment]
@FranzYuri just following-up, it's your turn,
please let me know if you need any help ;)

@Projeto-Jupiter/public-relations-outreach , tests are failling, doeas "build errored" mean that the problem is regarding the constructor?

**Comment by giovaniceotto** [comment]
In the test log, available here, we can see what the build error actually was: https://app.travis-ci.com/github/Projeto-Jupiter/RocketPy/builds/242391123

image

This means the style isn't following black. So we need to run black and commit again.

**Comment by Gui-FernandesBR** [comment]
Tks Gio!!

@FranzYuri it's a simple formatting problem, do you remeber how to run black foratting? If not please contact us or the @Projeto-Jupiter/back-end team and we definitely can help

File: rocketpy/Parachute.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        CdS : float
+            CdS of the parachute.
+        Trigger : function
+            Trigger function.
+        samplngRate : float
+            Sampling rate, in hertz, for the Trigger function.
+        lag : float
+            Time, in seconds, between the parachute ejection system is triggered and the
+        parachute is fully opened.
+        noise : tuple, list, optional
```
Comment (PatrickSampaioUSP) [inline rocketpy/Parachute.py:108]: Shouldn't the noise be optional? I think it is missing a:

`noise=(0, 0, 0)`

**Comment by FranzYuri** [comment]
Of course. But about that, I had already talked with Kierulff about how for
some reason even running black.py this error is occurring. I will run it
again and see if this issue is still occurring.

**Comment by FranzYuri** [comment]
Indeed, thank you.

**Review by Gui-FernandesBR — APPROVED** [review]
For me it's already good enough to be merged!!

@PatrickSampaioUSP can you re-review this one and then merge if ready? 
tks!

btw great job @FranzYuri , I hope this starts a good development in terms of recovery on RocketPy

**Pull request merged**